umh-core/pkg/control/loop.go

```diff
@@ -189,6 +190,10 @@ func (c *ControlLoop) Execute(ctx context.Context) error {
 				if errors.Is(err, context.DeadlineExceeded) {
 					// For timeouts, log warning but continue
 					sentry.ReportIssuef(sentry.IssueTypeWarning, c.logger, "Control loop reconcile timed out: %v", err)
+				} else if strings.Contains(err.Error(), "invalid desired state") {
```

Comment:
can you in the sources maybe create an error for it, and then match it here? then it becomes easier to find where the invalid desired state is coming from